enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -59,6 +59,7 @@ import (
 
 var firecrackerMountWorkspaceFile = flag.Bool("executor.firecracker_mount_workspace_file", false, "Enables mounting workspace filesystem to improve performance of copying action outputs.")
 var firecrackerCgroupVersion = flag.String("executor.firecracker_cgroup_version", "1", "Specifies the cgroup version for firecracker to use.")
+var firecrackerKillHostOnFailure = flag.Bool("executor.firecracker_kill_host_on_failure", false, "Kills the host executor process if any command orchestrating or running Firecracker fails. Useful for capturing failures preemptively. WARNING: using this option MAY leave the host machine in an unhealthy state on Firecracker failure; some post-hoc cleanup may be necessary.")
```

Comment:
how about calling this executor.die_on_firecracker_failure instead? (Since the host machine is not actually killed)